src/signature.rs

```diff
@@ -139,6 +140,18 @@ impl<'a> Drop for Signature<'a> {
     }
 }
 
+impl<'a> fmt::Display for Signature<'a> {
+
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let _ = write!(f, "{} <{}>",
```

Comment:
This should proxy along the return value of `write!` instead of explicitly ignoring it, but other than that this looks great to me, thanks!